src/ActionModal/components/UndelegationModal.vue

```diff
@@ -411,7 +411,11 @@ export default {
       `,
       /* istanbul ignore next */
       skip() {
-        return !this.address ||  !this.$refs.actionModal || !this.$refs.actionModal.show
+        return (
+          !this.address ||
+          !this.$refs.actionModal ||
+          !this.$refs.actionModal?.show
```

Comment:
```suggestion
          !this.$refs.actionModal?.show
```

the chaining does exactly reduce the need for that line

Reply:
true xDDD